docs/sources/source/speaker_recognition/quartznet.rst

```diff
@@ -0,0 +1,40 @@
+QuartzNet
+---------
+
+QuartzNet is a version of Jasper model with separable convolutions and larger filters. It can achieve performance
+similar to Jasper but with an order of magnitude less parameters.
+Similarly to Jasper, QuartzNet family of models are denoted as QuartzNet_[BxR] where B is the number of blocks, and R - the number of convolutional sub-blocks within a block. Each sub-block contains a 
+1-D *separable* convolution, batch normalization, ReLU, and dropout:
+
+We use Quarznet encoder of 3x2 size with narrow filters. These encoder is connected to decoder by using statspooling layer. 
+We experimented with various layers like gram layer, x-vector pooling, super vector which is combination of gram and x-vector. 
+Mean and variance based pooling is faster to train and very stable.
```

Comment:
You never mentioned what a mean and variance based pooling means.

Reply:
Done